docs/demo/demo.rst

```diff
@@ -10,8 +10,6 @@
 Paragraph Level Markup
 **********************
 
-.. contents:: Table of Contents
```

Comment:
Also removed several of those in page table of contents. Since out theme has the same content in the right sidebar, this is not needed (+ it makes all titles into links ..)